src/components/o-policy-features/index.js

```diff
@@ -0,0 +1,37 @@
+import classnames from 'classnames';
+import BaseComponentGlobal from '../../js/abstract/base-component-global';
+import defineOnce from '../../js/define-once';
+import styles from './index.scss';
+import template from './_template';
+import getAttribute from '../../js/get-attribute';
+
+class AXAPolicyFeatures extends BaseComponentGlobal {
+  static tagName = 'axa-policy-features'
+
+  static get observedAttributes() { return ['axa-style', 'title']; }
+
+  constructor() {
+    super({
+      styles, template,
+    });
+  }
+
+  connectedCallback() {
+    super.connectedCallback();
+    const axaStyle = getAttribute(this, 'axa-style');
```

Comment:
this logic here should be in template.js like https://github.com/axa-ch/patterns-library/blob/develop/src/components/m-button/_template.js

Reason why: connected callback is called AFTER the element is in the DOM. Template instead is bundled before. This can lead to unexpected results.